projects/packages/publicize/src/class-publicize-script-data.php

```diff
@@ -263,7 +263,7 @@ public static function get_api_paths() {
 		}
 
 		return array(
-			'refreshConnections' => '/jetpack/v4/publicize/connections?test_connections=1',
+			'refreshConnections' => '/wpcom/v2/publicize/connection-test-results',
```

Comment:
This won't work in Social plugin.

Reply:
IMHO #40607 implementation of moving the endpoint is better and will work in all scenarios.

It also has a well defined schema for the fields and also the properties marked as deprecated in the schema.

Reply:
Following that, we can deprecate the test connections endpoint because the above implementation incorporates that via a flag `test_connections` and we don't need to have a separate endpoint for that.

Reply:
> This won't work in Social plugin.

No, I know it went yet, as we also need to move the endpoint, but I've refactored the endpoints in this PR to use `get_all_connections_for_user` so everything is using the same code to prodcue the connections data